### PR TITLE
[zh-cn]: update the translation of CSS `outline-width` property

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/outline-width
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/outline-width/index.md
* Last commit: https://github.com/mdn/content/commit/aa714bb37625b21b0f40db1f1ea557e773456fa2

### CODE REVIEW

File: files/zh-cn/web/css/outline-width/index.md [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
 
-### Values
+### 值
 
 - {{cssxref("&lt;length&gt;")}}
-  - : The width of the outline specified as a `<length>`.
+  - : 轮廓的宽度指定为 `<length>`.
 - `thin`
-  - : 取决于用户代理。通常等同于桌面浏览器的 `1px` （包括 Firefox）。
+  - : 取决于用户代理。通常在桌面浏览器中等同于 `1px`（包括 Firefox）。
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-width/index.md:43]:
```suggestion
  - : 取决于用户代理。通常等同于桌面浏览器的 `1px`（包括 Firefox）。
```

File: files/zh-cn/web/css/outline-width/index.md [hunk 2]
```diff
@@ -1,63 +1,75 @@
...
 - `medium`
-  - : 取决于用户代理。通常等同于桌面浏览器的 `3px` （包括 Firefox）。
+  - : 取决于用户代理。通常在桌面浏览器中等同于 `3px`（包括 Firefox）。
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-width/index.md:45]:
```suggestion
  - : 取决于用户代理。通常等同于桌面浏览器的 `3px`（包括 Firefox）。
```

File: files/zh-cn/web/css/outline-width/index.md [hunk 3]
```diff
@@ -1,63 +1,75 @@
...
 - `thick`
-  - : 取决于用户代理。通常等同于桌面浏览器的 `5px`（包括 Firefox）。
+  - : 取决于用户代理。通常在桌面浏览器中等同于 `5px`（包括 Firefox）。
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-width/index.md:47]:
```suggestion
  - : 取决于用户代理。通常等同于桌面浏览器的 `5px`（包括 Firefox）。
```

File: files/zh-cn/web/css/outline-width/index.md [hunk 4]
```diff
@@ -1,63 +1,75 @@
...
+/* 全局值 */
 outline-width: inherit;
+outline-width: initial;
+outline-width: revert;
+outline-width: revert-layer;
+outline-width: unset;
 ```
 
-`outline-width` 的值可以是下面列表中的任一项。
+`outline-width` 属性的值可以是以下列出的任意一个。
```
Comment (T34-active) [inline files/zh-cn/web/css/outline-width/index.md:36]:
```suggestion
`outline-width` 属性的值可以是以下列表的任意一项。
```

**Review by T34-active — COMMENTED** [review]
nits

File: files/zh-cn/web/css/outline-width/index.md [hunk 5]
```diff
@@ -1,63 +1,75 @@
...
 
-`outline-width` 的值可以是下面列表中的任一项。
+`outline-width` 属性的值可以是以下列表的任意一项。
 
-### Values
+### 值
 
 - {{cssxref("&lt;length&gt;")}}
-  - : The width of the outline specified as a `<length>`.
+  - : 轮廓的宽度指定为 `<length>`.
```
Comment (yin1999) [inline files/zh-cn/web/css/outline-width/index.md:41]:
```suggestion
  - : 轮廓的宽度指定为 `<length>`。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**